packages/node/index.js

```diff
@@ -27,19 +31,59 @@ function patchResponse(res) {
   };
 }
 
+async function getProjectBaseUrl(encodedApiKey) {
+  const cacheDir = findCacheDir({ name: pkg.name, create: true });
+  const fsSafeApikey = crypto.createHash('md5').update(encodedApiKey).digest('hex');
+
+  // Since we might have differences of cache management, set the package version into the cache key so all caches will
+  // automatically get refreshed when the package is updated/installed.
+  const cacheKey = `${pkg.name}-${pkg.version}-${fsSafeApikey}`;
+
+  const cache = flatCache.load(cacheKey, cacheDir);
+
+  // Does the cache exist? If it doesn't, let's fill it. If it does, let's see if it's stale. Caches should have a TTL
+  // of 1 day.
+  const lastUpdated = cache.getKey('lastUpdated');
```

Comment:
I'm curious... what does the file on disk look like?

Reply:
Like this:

```
$ cat ./node_modules/.cache/readmeio/readmeio-3.0.0-099b0379a80334638e8a720ced83a94c
[{"lastUpdated":1595376648,"baseUrl":"1"},"https://docs.example.com"]
```

Looking at that I thought it was off, but it seems that the integer for `baseUrl` is the position in the array, as shown by adding more data into the cache:

```
[{"lastUpdated":1595376842,"baseUrl":"1","more data":"2"},"https://docs.example.com","jfefehfueufruewfr"]
```

Kind of weird, but it works!